packages/quantic/cypress/e2e/default-1/tab/tab-actions.ts

```diff
@@ -10,12 +10,15 @@ function tabActions(selector: TabSelector) {
       }
     },
     findTabPressTabPressSpace: (value: string) => {
-      // .tab() is equivalent to pressing the Tab; .type(' ') is equivalent to pressing the space bar;
-      selector.tab().contains(value).tab().type(' ');
+      // This is equivalent to pressing the Tab key twice and then pressing the Space bar.
+      selector.tab().contains(value).realPress(['Tab', 'Tab', 'Space']);
```

Comment:
why are we keeping `selector.tab()` here? isn't replaced by `.realPress(['Tab', 'Tab', 'Space'])`

Reply:
Ah my bad never mind my comment you can discard it, we need `selector.tab()` cause this not from the plugging, this is just the name of the selector defined in `tab-selectors.ts`

Reply:
yeah strangely the tests were still passing but I think I had a typo. Anyways yeah you are right it is quite confusing since they have the same name. Now, hopefully it will be less so with the `tab()` and `.realPress('Tab')`